Libraries/Components/View/ViewPropTypes.js

```diff
@@ -100,22 +100,64 @@ type DirectEventProps = $ReadOnly<{|
 |}>;
 
 // [macOS
-type KeyboardEventProps = $ReadOnly<{|
+/**
+ * Represents a key that could be passed to `validKeysDown` and `validKeysUp`.
+ *
+ * `key` is the actual key, such as "a", or one of the special values:
+ * "Tab", "Escape", "Enter", "ArrowLeft", "ArrowRight", "ArrowUp", "ArrowDown",
+ * "Backspace", "Delete", "Home", "End", "PageUp", "PageDown".
+ *
+ * The rest are modifiers that when absent mean don't care (either state -- true or false --
```

Comment:
I think the absent modifier bit, we can omit from the PR as well. It's not part of the [keyboard spec](https://github.com/microsoft/react-native-windows/blob/main/vnext/proposals/active/keyboard-reconcile-desktop.md) and I considered the previous behavior a shortcoming (the fact that if `validKeysDown` specified "Enter", it would also block `Cmd+Enter`). We've also seen bugs related to that on the [FURN](https://github.com/microsoft/fluentui-react-native/blob/438bde370a21abd4e4750f99915fc8e9ec987370/packages/utils/interactive-hooks/src/useKeyProps.ts#L16C6-L16C6) side to account for modifiers. I'd rather keep this consistent across platforms, even if we're not aligning the API just yet.

Reply:
I'll make it consistent with the keyboard spec (i.e. omitting it means "false", not "don't care"). However, for now, I think I'll limit it to the new form of event specification (e.g. `{key: 'Foo'}`, not `'Foo'`), to keep backward compatibility with the older API spec for the Mac.